css/index.css

```diff
@@ -1,3 +1,529 @@
+}
+
+.nav a:hover, a:visited, a:link, a:active {
+    text-decoration: none;
+    color: darkslategrey;
+}
+
+nav .logo {
+    width: 183px;
+    margin: 10px;
+	justify-content: flex-end;
+}
+
+header {
+	width: 900px;
+	display: flex;
+	justify-content: flex-end;
+    align-items: center;
+    margin: 42px 8px 40px 0;
+    padding-bottom: 8px;
+}
+
+header .getting-started {
+    width: 449px;
+    margin-top: 7px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    text-align: center;
+}
+
+header .getting-started .start {
+    font-size: 10;
+    margin: 14px 10px;
+	padding: 8px 45px;
+	border: 1px solid black;
+}
+
+header .snippet {
+	width: 380px;
+	margin: 20px 0 25px 3px;
+	justify-content: flex-end;
+}
+
+main {
+	width: 900px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+}
+
+main .top {
+    width: 879px;
+    margin: 0 auto;
+	display: flex;
+}
+
+.features-and-about p {
+    margin: 22px 0;
+    font-size: 17px;
+}
+
+main .top .features-and-about {
+    margin: auto;
+    padding-top: 36px;
+    border-top: 1px solid black;
+	justify-content: flex-start;
+}
+
+main .middle {
+    width: 879px;
+    margin: auto;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+main .middle .middle-img {
+    width: 877px;
+    margin-top: 10.5px;
+}
+
+main .bottom {
+    width: 880px;
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	flex-wrap: wrap;
+	padding-bottom: 37px;
+}
+
+main .bottom .services, .product, .vision {
+    width: 274px;
+    margin-top: 32px;
+    margin-left: 3px;
+    justify-content: flex-start;
+    line-height: 1.1;
+}
+
+.contact {
+	width: 880px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+    border-top: 1px solid black;
+}
+
+.contact h2 {
+    margin-top: 36px;
+}
+
+address p {
+    padding: 2px 0 18px 0;
+}
+
+.contact p {
+    padding: 1px 0 15px 0;
+}
+
+footer {
+    width: 900px;
+    margin-top: 26px;
+	display: flex;
+	justify-content: center;
+}
+
+/* ========================================= tablet */
+
+@media (max-width: 800px) {
+    h2 {
+        font-size: 16px;
+    }
+    
+    p {
+        font-size: 16px;
+    }
+
+    .container {
+        width: 780px;
+        margin: 20px 0;
+        display: flex;
+        flex-direction: column;
+        flex-wrap: wrap;
+        align-content: center;
+    }
+
+    nav {
+        width: 760px;
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    nav .nav-items {
+        width: 700px;
+        margin-top: 30px;
+        margin-left: 16px;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    nav .logo {
+        width: 183px;
+        margin: 10px 0 10px 40px;
+        align-items: center;
```

Comment:
It's my understanding that `align-items` only applies to elements that are flex parents (ie. have `display: flex;` as one of their styles).